src/cmd/group.go

```diff
@@ -38,9 +39,161 @@ EOF
 	},
 }
 
-// TODO: Get Group
+var getGroupCommand = &cobra.Command{
+	Use:        "group ID|ALIAS",
+	Short:      "Get details about a group",
+	Long:       `Get details about a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		common.PrettyPrint(group)
+		common.WasFound(group.Id == nil, key)
+	},
+}
 
-// TODO: List Groups
+var getMembersCommand = &cobra.Command{
+	Use:        "members ID|ALIAS",
+	Short:      "Get members for a group",
+	Long:       `Get members for a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		members, err := group.Members(getClientGQL())
+		cobra.CheckErr(err)
+		common.PrettyPrint(members)
+		common.WasFound(group.Id == nil, key)
+	},
+}
+
+var getDescendantRepositoriesCommand = &cobra.Command{
+	Use:        "repositories ID|ALIAS",
+	Short:      "Get descendant repositories for a group",
+	Long:       `Get descendant repositories for a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
```

Comment:
Since this is repeated more then 3 times it should be pull out to a reusable func

Reply:
Yeah, and I got it from somewhere else so I'm sure it's the same case for other get commands